.gitignore

```diff
@@ -66,6 +66,9 @@ node_modules/
 codekit-config.json
 .pycharm_helpers/
 
+.project
+.pydevproject
```

Comment:
The default one from github adds tons of that stuff, and then it looks like we snuck in some eclipse stuff.  I say go ahead and leave this in and we can address it all later as an extremely low priority issue lol